packages/web-frontend/cypress.json

```diff
@@ -3,8 +3,7 @@
   "testFiles": "*.spec.js",
   "integrationFolder": "cypress/e2e",
   "projectId": "pty5cc",
-  "requestTimeout": 30000,
-  "responseTimeout": 60000,
```

Comment:
No responseTimeout? I guess requestTimeout covers all we need?

Reply:
The [default responseTimeout](https://docs.cypress.io/guides/references/configuration#Timeouts) of 30secs will be used, it seems that this is enough